test/functional/cmdLineTests/loadLibraryTest/loadLegacyLibraryTest.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no" ?>
```

Comment:
Can you write this to use TestNG rather than commandline tester?  The Commandline tester framework is a legacy approach and we generally prefer to use TestNG or alternatives to commandline tester

Reply:
changed to use TestNG framework